### PR TITLE
@alloy => Updated gene and tag

### PR DESCRIPTION
**Pull request by mzikherman**
This works with the new Reaction components we'll publish for Relay modern.

### CODE REVIEW

File: desktop/apps/gene/client.js [hunk 1]
```diff
@@ -19,17 +23,39 @@ const relatedArtistsTemplate = (args) => {
...
+    mode = sd.MODE
+  }
 
-  init({
-    user: sd.CURRENT_USER,
-    component: components.Gene,
-    domID: 'gene-filter',
-    queryConfig: new configs.GeneQueryConfig(options)
-  })
+  const options = Object.assign({}, { sort }, { filters: { ...urlParams } }, { geneID: sd.GENE.id, mode })
```
Comment (alloy) [inline desktop/apps/gene/client.js]: Why do you use `Object.assign(…)` here? Can’t you just create 1 object immediately instead of merging?

Comment (mzikherman) [reply desktop/apps/gene/client.js]: I'm using `Object.assign` to create the new object....I've seen this pattern everywhere.

Comment (alloy) [reply desktop/apps/gene/client.js]: Merging obviously definitely has its place, but here you are creating 4 new objects and then merging them. As far as I can tell this is the equivalent of:

```js
 { sort, filters: { ...urlParams }, geneID: sd.GENE.id, mode }
```

Am I overlooking/misunderstanding something?

Comment (mzikherman) [reply desktop/apps/gene/client.js]: No. My point was just that I've noticed we use this pattern (`assign`), all over, even in places where I would agree with you, it just seems like we're creating a new object.

So I just decided to follow the existing pattern. It doesn't matter to me, I can switch.

Comment (alloy) [reply desktop/apps/gene/client.js]: I think the code is simpler to read without it. And while a single instance won’t be too bad, if this is really happening all over the place then at some point the overhead of all these objects will add up.

**Pull request merged**